src/controllers/url.ts

```diff
@@ -0,0 +1,86 @@
+import { Request, Response } from 'express';
+import { customAlphabet } from 'nanoid';
+import Url from '../models/url';
+import { BASE_URL, HTTP_ERRORS } from '../constant';
+import logger from '../logger/index';
+
+const createShortUrls = async (req: Request, res: Response) => {
+  try {
+    const { url, customName } = req.body;
+    let shortUrls;
+    const generateShortId = customAlphabet(
+      'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890',
+      5
+    );
+
+    if (!url) {
+      return res.status(HTTP_ERRORS.BAD_REQUEST).json({ error: 'Original Url is required' });
+    }
+    const existingLongUrl = await Url.findOne({ originalUrl: url });
+    if (existingLongUrl) {
+      return res
+        .status(200)
```

Comment:
Should be `BAD_REQUEST` or `CONFLICT` and should use a Variable

Reply:
This is done. Just replying in case I don't push this night so I can at least have some contribution or activity today